src/lib/meilisearch.ts

```diff
@@ -20,12 +20,14 @@ import {
   EnqueuedDump,
 } from '../types'
 import { HttpRequests } from './http-requests'
+import { addProtocolIfNotPresent } from './utils'
 
 class MeiliSearch {
   config: Config
   httpRequest: HttpRequests
 
   constructor(config: Config) {
+    config = addProtocolIfNotPresent(config)
```

Comment:
to be consistent with the line underneath 
```suggestion
    config.host = addProtocolIfNotPresent(config.host)
```